cloud-sql/sql-server/sqlalchemy/connect_connector.py

```diff
@@ -45,6 +51,9 @@ def getconn() -> pytds.Connection:
             user=db_user,
             password=db_pass,
             db=db_name,
+            # [START_EXCLUDE]
+            cafile=db_root_cert,
```

Comment:
```suggestion
            cafile=db_root_cert,
            validate_host=False,
```
does this work without setting `validate_host=False`?